src/www/app/util.ts

```diff
@@ -12,13 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import * as errors from '../model/errors';
+import * as errors from "../model/errors";
```

Comment:
nit (prettier fail): use single quotes here and in the next line.

Reply:
Done in ae44e1e